packages/style-guide/eslint/rules/next.js

```diff
@@ -10,5 +11,11 @@ export default [
       ...pluginNext.configs.recommended.rules,
       ...pluginNext.configs['core-web-vitals'].rules,
     },
+    languageOptions: {
+      globals: {
+        ...globals.browser,
```

Comment:
thought: `...globals.browser,` is already set up by `react.js`, dunno if it should be included twice.